cmd/go-filecoin/init.go

```diff
@@ -43,9 +44,11 @@ var initCmd = &cmds.Command{
 	Options: []cmdkit.Option{
 		cmdkit.StringOption(GenesisFile, "path of file or HTTP(S) URL containing archive of genesis block DAG data"),
 		cmdkit.StringOption(PeerKeyFile, "path of file containing key to use for new node's libp2p identity"),
+		cmdkit.StringOption(WalletKeyFile, "path of file containing keys to import into the wallet on initialization"),
```

Comment:
These two options are required to set up a bootstrap miner with no extra configuration between `init` and `daemon`